tests/test_blocking_network.py

```diff
@@ -129,6 +151,27 @@ def test_no_vcr_mark(httpbin):
     result.assert_outcomes(passed=2)
 
 
+def test_block_network_via_cmd_with_whitelist(testdir):
```

Comment:
This one seems almost identical to the previous one - can we use parametrize? E.g parametrize args to `runpytest` call?